package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@bigcommerce/stencil-utils",
-  "version": "6.15.0",
+  "version": "6.16.0",
```

fix is a minor change, following semver the last number should be bumped
```suggestion
  "version": "6.15.1",
```

---

Done ✅